doc/authors.rst

```diff
@@ -93,12 +93,16 @@
     <p>Brian Holt</p>
     </td>
     <td>
-    <a href='https://github.com/arjoly'><img src='https://avatars0.githubusercontent.com/u/1274722?v=4' class='avatar' /></a> <br />
-    <p>Arnaud Joly</p>
+    <a href='https://github.com/adrinjalali'><img src='https://avatars3.githubusercontent.com/u/1663864?v=4' class='avatar' /></a> <br />
+    <p>Adrin Jalali</p>
     </td>
     </tr>
     <tr>
     <td>
+    <a href='https://github.com/arjoly'><img src='https://avatars0.githubusercontent.com/u/1274722?v=4' class='avatar' /></a> <br />
+    <p>Arnaud Joly</p>
```

Comment:
I don't know why Arnaud was missing before?

Reply:
Arnaud was not missing, Adrin took his place above, it is just a diff mistake.